tests/integration/ambient/waypoint_test.go

```diff
@@ -381,21 +381,33 @@ func TestWaypointDNS(t *testing.T) {
 				if src.Config().HasSidecar() {
 					t.Skip("TODO: sidecars don't properly handle use-waypoint")
 				}
-				address := "240.240.240.239"
-				if _, v6 := getSupportedIPFamilies(t); v6 {
-					address = "2001:2::f0f0:239"
+				v4, v6 := getSupportedIPFamilies(t)
+				if v4 {
+					src.CallOrFail(t, echo.CallOptions{
```

Comment:
Long term, what we may want to do is add an option into echo.CallOptions which is either (or both):

* `Addresses []string`: call each one
* DualStack=true: call the address, but do a dns lookup only for V4 then only for V6.

With the auto allocation v2 work @ilrudie is doing/just merged, we can actually call the hostname here and do the DNS lookup.

Then its simple to call both

Reply:
Not for this PR, just long term -- ideally we get to the point we are doing dual stack in every test, so making it simple is probably increasingly the likelihood it actually gets tested